src/test/scala/org/apache/spark/deploy/armada/ConfigSuite.scala

```diff
@@ -34,38 +30,36 @@ class ConfigSuite
    with TableDrivenPropertyChecks
    with Matchers {
   test("testClusterSelectorsValidator") {
-    case class TestCase(
-      testSelectors: String,
-      expectedValid: Boolean,
-      name: String)
-
-    val testCases = List[TestCase](
+    val testCases = Table(
```

Comment:
In other tests of this Suite we use `Table`, we should stick to one approach of test parametrization.